manta-pay/src/crypto/ecc/arkworks.rs

```diff
@@ -363,6 +364,7 @@ where
 }
 
 /// Elliptic Curve Group Element Variable
+#[derive(Clone)]
```

Comment:
Please use `derivative::Derivative` for `derive` traits when there are type parameters on the deriving type so we have the correct semantics.

Reply:
done